recipes/php.rb

```diff
@@ -18,24 +18,10 @@
   action :nothing
 end
 
-probe_version = node[cookbook_name]['php']['version'] ? node[cookbook_name]['php']['version'] : Blackfire::Versions.probe(node)
-
+probe_version = Blackfire::Versions.probe(node)
```

Comment:
this change disallows one to pin which blackfire package version they want to install

Reply:
Ho, I just understand usage of this attribute. I believed that was to set PHP version...
I will readd that.

Reply:
PR Updated